src/diagonal.works/b6/api/functions/geojson_test.go

```diff
@@ -1,16 +1,32 @@
 package functions
 
 import (
+	"fmt"
 	"testing"
 
 	"diagonal.works/b6"
 	"diagonal.works/b6/api"
 	"diagonal.works/b6/geojson"
+	"diagonal.works/b6/ingest"
 	"diagonal.works/b6/test/camden"
 )
 
-func TestGeoJSON(t *testing.T) {
-	granarySquare := camden.BuildGranarySquareForTests(t)
+func TestGenericFlowGeoJSON(t *testing.T) {
```

Comment:
I'm guess the idea is we rip all of the bits that turn on the generic flow in the tests out when we're comfortable of switching to the generic flow?

Reply:
yes i love work